cli/src/feature.rs

```diff
@@ -344,7 +344,7 @@ fn feature_activation_allowed(rpc_client: &RpcClient, quiet: bool) -> Result<boo
         )
         .unwrap_or((false, false));
 
-    if !stake_allowed && !rpc_allowed && !quiet {
+    if !(quiet || stake_allowed && rpc_allowed) {
```

Comment:
```suggestion
    if !quiet || !(stake_allowed && rpc_allowed) {
```
Wdyt? I think it would be helpful to see the feature-version info on all `!quiet` calls.

Reply:
How about just `!quiet` in that case?

Reply:
Honestly, I was thinking maybe remove the if clause altogether and always print it...

Reply:
On a slight tangent, I'm surprised no one has asked for json output here yet.

Reply:
> Honestly, I was thinking maybe remove the if clause altogether and always print it...

I figure we respect `--quiet`

> On a slight tangent, I'm surprised no one has asked for json output here yet.

Triton may have brought it up as a nice-to-have at some point :sweat_smile: